Samples/Swift/nhubsample/nhubsample/Constants.swift

```diff
@@ -0,0 +1,12 @@
+//
+//  Copyright © 2018 Microsoft All rights reserved.
+//  Licensed under the Apache License (2.0).
+//
+
+import Foundation
+
+struct Constants {
```

Comment:
User enums for constants to give them a namespace.

Reply:
So usage would be something like:
`Constants.ConnectionString`
?

Reply:
It would be like:
`Constants.ConnectionString. rawValue `
But in our case we will use only `rawValue`, so why we need to use a `enums`?

Reply:
I'm fine with sticking with the current approach. This seems like something we can cleanup after the initial PR.